docs/content/how-do-i-write-policies.md

```diff
@@ -324,30 +331,29 @@ cannot refer to the index of an element within a set.
 
 When comparing sets, the order of elements does not matter:
 
+```live:eg/set_equality:query:merge_down
+{1,2,3} == {3,1,2}
 ```
-> {1,2,3} == {3,1,2}
-true
+```live:eg/set_equality:output
 ```
 
 Because sets are unordered, variables inside sets must be unified with a ground
 value outside of the set. If the variable is not unified with a ground value
 outside the set, OPA will complain:
 
-```ruby
-> {1,2,3} == {3,x,2}
-1 error occurred: 1:1: rego_unsafe_var_error: var x is unsafe
+```live:eg/set_unification:query:merge_down
+{1,2,3} == {3,x,2}
+```
+```live:eg/set_unification:output:expect_unsafe_var
 ```
 
 Because sets share curly-brace syntax with objects, and an empty object is
 defined with `{}`, an empty set has to be constructed with a different syntax:
 
-```ruby
-> n := count(set())
-+---+
-| n |
-+---+
-| 0 |
-+---+
+```live:eg/set_construction:module:merge_down
+n = count(set())
```

Comment:
Same, except to avoid declaring a rule in the REPL we ought to just write `count(set())` here.